root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -917,6 +917,17 @@ limited_link_type_combinations: [
             expected_clean_url: 'https://gamechops.bandcamp.com/campaign/samus-chill',
        only_valid_entity_types: ['release'],
   },
+  {
+                     input_url: 'https://bandcamp.com/videoframe?video_id=188124/',
+             input_entity_type: 'recording',
+    expected_relationship_type: undefined,
+       input_relationship_type: 'streamingfree',
+       only_valid_entity_types: [],
+                expected_error: {
+                                  error: 'do not link to /videoframe links',
```

Comment:
To match the suggestion in the previous comment:

```suggestion
                                  error: 'Please do not add',
```